doc/man1/fipsinstall.pod

```diff
@@ -0,0 +1,148 @@
+=pod
+
+=head1 NAME
+
+openssl-fipsinstall,
+fipsinstall - perform FIPS configuration installation
+
+=head1 SYNOPSIS
+
+B<openssl fipsinstall>
+[B<-help>]
+[B<-in configfilename>]
+[B<-out configfilename>]
+[B<-module modulefilename>]
+[B<-provider_name providername>]
+[B<-section_name sectionname>]
+[B<-verify>]
+[B<-mac_name macname>]
+[B<-macopt>]
+
+B<openssl> I<fipsinstall> [B<...>]
+
+=head1 DESCRIPTION
+
+During FIPS installation this utility is used to run the FIPS modules self tests
+and then generate a FIPS related configuration file that contains:
```

Comment:
I think this needs to be rephrased slightly, so it's telling the users how it's useful for them, rather than telling them how we use it.  I assume most of the words will be the same, just phrased differently.

Reply:
I assume this utility is going to be used as part of the build process?

Reply:
Yes, but the manual is for external users, for *their* use of this command.  They will be interested in what the use is for them, not in the story of how we use it.  Do you understand the difference?

Reply:
(well, the manual is for us as well, but can be written for the external party and still be useful for us as well)

Reply:
yep - just changed the tense..